app/src/main/java/org/simple/clinic/contactpatient/ContactPatientModel.kt

```diff
@@ -103,4 +103,8 @@ data class ContactPatientModel(
   fun contactPatientInfoLoading(): ContactPatientModel {
```

Comment:
Can we update the commit message to "When current facility is loaded, then update the model"? The current commit message doesn't properly indicate that.